README.md

```diff
@@ -1,50 +1,166 @@
+  </h3>
+  <div>
 
-# The project
+  <a href="">[![Issues](https://img.shields.io/github/issues/webdev-20/shorti-url-shortener.svg?style=for-the-badge)](https://github.com/webdev-20/shorti-url-shortener/issues)</a>
+  <a href="">[![contributors](https://img.shields.io/github/contributors/webdev-20/shorti-url-shortener.svg?style=for-the-badge)](https://github.com/webdev-20/shorti-url-shortener/contributors)</a>
+  <a href="">[![Forked](https://img.shields.io/github/forks/webdev-20/shorti-url-shortener.svg?style=for-the-badge)](https://github.com/webdev-20/shorti-url-shortener)</a>
 
-URL shortener 
+  </div>
+</div>
 
-## Tech Stack
 
-- React
-- CSS Modules
-- Mongo
-- Express
+# The Lassie Lounge URL Shortner
 
-## Features 
-please check the pinned issue
+This project helps user to generate a short URL making it easy to share and use.
+<a name = "readme-top"></a>
 
-## Live site
+## Table of Contents
+  
+  1. [About The Project](#about-the-project)
+     - [Built With](#built-with)
+  2. [Getting Started](#getting-started)
+       - [Prerequisites](#prerequisites)
+       - [Run project locally](#run-project-locally)
+  3. [Contributing](#contributing)
+  4. [API Documentation](#api-documentation)
+  5. [Features](#features)
+  6. [Live Site](#live-site)
+  7. [License](#license)
+  8. [Contacts](#contacts)
+  9. [Acknowledgements](#acknowledgements)
+
+
+
+## About The Project
+
+Shorti URL Shortener allows user to shorten their URL using this API.
+No API keys are required to create an account with us.
+
+Designed with 2 endpoints - **Authentication** and **Links** on CRUD methods.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+  
+  ## Built With
+
+  **Client:** 
+
+  <a href="">[![React](https://img.shields.io/badge/React-20232A?style=for-the-badge&logo=react&logoColor=61DAFB)](https://reactjs.org])</a>
+
+  <a href="">![CSS Modules](https://img.shields.io/badge/CSS%20Modules-000000?style=for-the-badge&logo=CSS-Module&logoColor=white)</a>
+
+
+**Server:** 
+
+<a href="">[![Express](https://img.shields.io/badge/Express%20Js-yellow?style=for-the-badge&logo=expressJS)](https://expressjs.com/)</a>
+
+
+<a href="">[![Mongo](https://img.shields.io/badge/MongoDB-success?style=for-the-badge&logo=mongodb&logoColor=white&style=flat)](https://www.mongodb.com/)
+
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+## Getting Started
+
+  > To learn how to set up the project locally, follow these simple steps below:
+
+### Prerequisites:
+
+  > Should have **Node.js** installed to get yarn package.
+
+### Run project locally
+
+1. Fork and Clone the repository
+
+```bash
+  git clone https://github.com/webdev-20/shorti-url-shortener.git
+```
+
+2. Go to the project directory
+
+```bash
+  cd shorti-url-shortener
+```
+
+3. Install dependencies in both client and server
+
+```bash
+  yarn install
+```
+
+4. Start the server
+
+```bash
+  yarn start  ( for both client and server)
```

Comment:
formatting is inconsistent with the brackets, some has space and some has no space